lib/liquid/translators/tags/custom_tag.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Liquid.Translators.Tags.CustomTag do
+  alias Liquid.{Template, Tag}
+
+  def translate(custom_name: name, custom_markup: markup) do
+    tag_name = String.to_atom(name)
+    custom_tag = Application.get_env(:liquid, :extra_tags)
+
+    case is_map(custom_tag) do
```

Comment:
Its better to use `if/else` instead of a case, but since it is nested, it might be better to use a `cond` statement.

Reply:
Ok, will be refactored